app/src/components/Dropdown.tsx

```diff
@@ -0,0 +1,33 @@
+import { ChangeEvent, ReactElement } from 'react'
+import styled from 'styled-components'
+
+type Props = {
+  handleChange: (e: ChangeEvent<HTMLSelectElement>) => void
+  id: string
+  label: string
+  options: string[]
+}
+
+const Dropdown = (props: Props): ReactElement => {
+  const { handleChange, id, label, options } = props
+
+  return (
+    <form className="usa-form">
+      <Label className="usa-label" htmlFor={id}>
+        <b>{label}</b>
```

Comment:
```suggestion
        <strong>{label}</strong>
```

Let's use the more semantic `<strong>` tag over the less semantic `<b>` tag.